assets/css/components/post-block.scss

```
@@ -0,0 +1,136 @@
+.post-block {
+  position: relative;
+  display: grid;
+  grid-template-columns: 70% 30%;
+  align-items: center;
+  column-gap: rem(40px);
+
+  @include breakpoint($break: 'small', $dir: 'max-width') {
+    grid-template-columns: rem(150px) auto;
+    column-gap: rem(16px);
+  }
+
+  &__category {
+    grid-column: 1 / span 1;
+    grid-row: 1 / span 1;
+    margin-top: rem(32px);
+    color: $color-orange-180;
+    @extend %font-ui-text-base-oxygen;
+    font-weight: bold;
+    letter-spacing: 0.03em;
+    text-transform: uppercase;
+
+    @include breakpoint($break: 'small', $dir: 'max-width') {
+      grid-column: 1 / span 2;
+      margin-top: 0;
+    }
+  }
+
+  &__title {
+    grid-column: 1 / span 1;
+    grid-row: 2 / span 1;
+    margin-top: rem(4px);
+    color: $color-white-190;
+    @extend %font-ui-heading-post-block-title;
+    font-weight: 500;
+
+    @include breakpoint($break: 'small', $dir: 'max-width') {
+      grid-column: 1 / span 2;
+    }
+  }
+
+  &__author {
+    margin-left: rem(40px);
+
+    @include breakpoint($break: 'small', $dir: 'max-width') {
+      margin-top: rem(16px);
+      margin-left: 0;
+    }
+  }
+
+  &__excerpt {
+    grid-column: 1 / span 1;
+    grid-row: 4 / span 1;
+    margin: rem(17px) 0 rem(32px) 0;
+    color: $color-white-180;
+    @extend %font-ui-text-lg-oxygen;
+    line-height: 150%;
```

```suggestion
    line-height: 1.5;
```

Best practice is to use unitless line-heights so they scale.